.github/workflows/semver-require.yaml

```diff
@@ -0,0 +1,16 @@
+name: Pull Request Labels
+on:
+  pull_request:
+    types:
+      - opened
+      - labeled
+      - unlabeled
```

Comment:
The example at https://github.com/mheap/github-action-required-labels#complete-example also runs on `synchronize`.
What does it mean for us not to have the `synchronize` type? In particular, are we certain it runs after someone pushes a new commit to a PR?
Take this scenario for instance: someone opens a PR and does not set labels, this person then add a new commit. The `semver-unlabel` workflow won't do anything, and `semver-require` won't kick-in. Will the PR be allowed to merged without a label?

Reply:
nice catch!